Source/Header/DayHeaderView.swift

```diff
@@ -16,6 +22,14 @@ public final class DayHeaderView: UIView, DaySelectorDelegate, DayViewStateUpdat
     didSet {
       state?.subscribe(client: self)
       swipeLabelView.state = state
+        
+      // Fixes the "jump" of the pager from today to the selected date. This is noticeable
+      // when we are on a date outside of today's week, and try to change the time zone.
+      let previousSelectedDate = state!.selectedDate // day selected before timezone change
+      let vc = makeSelectorController(startDate: beginningOfWeek(previousSelectedDate))
+      vc.selectedDate = previousSelectedDate
+      currentWeekdayIndex = vc.selectedIndex
+      pagingViewController.setViewControllers([vc], direction: .forward, animated: false, completion: nil)
```

Comment:
I think, any glitches during the configuration process are OK, as it should happen "behind the scenes", i.e. when the   DayViewController is not displayed or obscured by another view.